unit-tests/run-unit-tests.py

```diff
@@ -421,11 +421,11 @@ def devices_by_test_config( test, exceptions ):
             continue
 
 
-def test_wrapper_( test, configuration=None, repetition=1, retry=0, sns=None ):
+def test_wrapper_( test, configuration=None, repetition=1, curr_retry=0, last_retry = 0, sns=None ):
```

Comment:
Maybe for being clearer we can change `last_retry` to `max_retries` ? or `retries_count`?

Reply:
Renamed